client/src/common/components/Members/components/MemberDetails.jsx

```diff
@@ -229,7 +217,7 @@ class MemberDetails extends PureComponent {
         </div>
         {isInfoVisible && (
           <p className="member-details__role-description">
-            {infoText[route][role]}
+            <FormattedMessage id={`common.member-details.${route}.${role}`} />
```

Comment:
please do not use id concatenation - there will be problem with finding and replacing such id